sanssouci/tests/test_lambda_calibration.py

```diff
@@ -33,6 +34,19 @@ def test_get_perm_p():
     assert np.sum(pvals < .1) < B * p * .12
 
 
+def test_get_permuted_p_values_one_sample():
+    rng = np.random.RandomState(42)
+    n = 20
+    p = 50
+    X = rng.randn(n, p)
+    B = 100
+    pvals = get_permuted_p_values_one_sample(X, B=B)
+    assert pvals.shape == (B, p)
+    assert pvals.min() > 1.e-7
+    assert pvals.max() <= 1
+    assert np.sum(pvals < .1) < B * p * .12
```

Comment:
If pvals are supposed to be sorted, you need to check that they are.

Reply:
Done, see https://github.com/alexblnn/sanssouci.python/blob/one_sample/sanssouci/tests/test_lambda_calibration.py#L45 (maybe there is a smarter way to do this...)